app/config/Module.scala

```diff
@@ -25,7 +25,12 @@ class Module extends AbstractModule {
 
   override def configure(): Unit = {
     bind(classOf[DataRequiredAction]).to(classOf[DataRequiredActionImpl]).asEagerSingleton()
-    bind(classOf[Clock]).toInstance(Clock.systemDefaultZone.withZone(ZoneOffset.UTC))
+    bind(classOf[Clock]).toInstance(
+      Clock.fixed(
+        Instant.parse("2023-10-01T12:00:00.00Z"),
+        ZoneId.of("Australia/Melbourne")
+      )
+    )
```

Comment:
revert the clock please

Reply:
This has slipped back in when trying to resolve a new GPG key issue with rebase -i. Will remove.